data/global/excel/uniqueitems.txt

```diff
@@ -1415,9 +1415,9 @@ Chipped Diamond Grabber	1410	0	1			1	1	1	1	stt	Chipped Diamond Grabber		1	1
 Chipped Emerald Grabber	1411	0	1			1	1	1	1	stt	Chipped Emerald Grabber		1	1																																																									0
 Chipped Skull Grabber	1412	0	1			1	1	1	1	stt	Chipped Skull Grabber		1	1																																																									0
 Chipped Ruby Grabber	1413	0	1			1	1	1	1	stt	Chipped Ruby Grabber		1	1																																																									0
-t4 Splash Charm	1414	100	0			1	1	1	24	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashtwo	proc_SplashDamage_Two	100	1	dmg2%		-30	-33																																										0
-t5 Splash Charm	1415	100	0			1	1	1	30	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashtwo	proc_SplashDamage_Two	100	1	dmg2%		-25	-30																																										0
-t6 Splash Charm	1416	100	0			1	1	1	45	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashthree	proc_SplashDamage_Three	100	1	dmg2%		-20	-27																																										0
-t7 Splash Charm	1417	100	0			1	1	1	60	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashthree	proc_SplashDamage_Three	100	1	dmg2%		-15	-25																																										0
-t8 Splash Charm	1418	100	0			1	1	1	75	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashthree	proc_SplashDamage_Three	100	1	dmg2%		-10	-23																																										0
-t9 Splash Charm	1419	100	1			1	1	1	90	cm5	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashfour	proc_SplashDamage_Four	100	1	dmg2%		-5	-20																																										0
+t4 Splash Charm	1414	100	0			1	1	1	24	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashtwo	proc_SplashDamage_Two	100	1	dmg2%		-15	-15																																										0
+t5 Splash Charm	1415	100	0			1	1	1	30	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashtwo	proc_SplashDamage_Two	100	1	dmg2%		-10	-10																																										0
+t6 Splash Charm	1416	100	0			1	1	1	45	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashthree	proc_SplashDamage_Three	100	1	dmg2%		-5	-5																																										0
+t7 Splash Charm	1417	100	0			1	1	1	60	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashthree	proc_SplashDamage_Three	100	1	dmg2%		1	1																																										0
+t8 Splash Charm	1418	100	0			1	1	1	75	cm4	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashthree	proc_SplashDamage_Three	100	1	dmg2%		2	3																																										0
+t9 Splash Charm	1419	100	1			1	1	1	90	cm5	charm	1	1	1			flpmss	invmss	item_gem	12	item_gem	splashfour	proc_SplashDamage_Four	100	1	dmg2%		3	5																																										0
```

Comment:
I think the new adjustments are a bit on the OP side. I like the old numbers more :/

Additionally it may be worth it to group the charms together in uniqueitems. Just easier that way.

Reply:
it wouldn't do a mess with something? just copy paste, change the ID's and delete the old rows?

Reply:
didn't get the fifth splash row to work, guess here is the "wall" for me - hope you have some time in the future to look at those charming charmys and test it

Reply:
needed to bring the t1-t3 charms back up, they messed the gem bag up again (assets, name gone).